cpp/oneapi/dal/util/load_graph_descriptor.hpp

```diff
@@ -14,19 +14,28 @@
 * limitations under the License.
 *******************************************************************************/
 
+/// @file
+/// Types and descriptors of the operations for graph loading functionality
+
 #pragma once
 
 #include "oneapi/dal/data/graph_common.hpp"
 #include "oneapi/dal/data/undirected_adjacency_array_graph.hpp"
 
 namespace oneapi::dal::preview::load_graph {
 
+/// Structure which defines the parameters of the operation of the graph loading
```

Comment:
**A structure,** which defines the parameters of the **graph loading** operation